avae/train.py

```diff
@@ -630,6 +629,17 @@ def train(
                 else:
                     ps = None
 
+            vis.latent_4enc_interpolate_plot(
```

Comment:
Since this has much overlap with the existing `vis.interpolation_plots` in future might be better to just make direct changes rather than create a new function. I'll make a PR with a merge of these two.

Reply:
Yep, I agree the interpolation is more clear here.